package/boot/uboot-envtools/files/ramips

```diff
@@ -57,13 +54,19 @@ linksys,ea7300-v1|\
 linksys,ea7300-v2|\
 linksys,ea7500-v2|\
 linksys,ea8100-v1|\
-linksys,ea8100-v2|\
+linksys,ea8100-v2)
+	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x20000"
+	;;
 xiaomi,mi-router-3g|\
+xiaomi,mi-router-3g-v2|\
 xiaomi,mi-router-3-pro|\
 xiaomi,mi-router-4|\
+xiaomi,mi-router-4a-gigabit|\
 xiaomi,mi-router-ac2100|\
+xiaomi,miwifi-3c|\
 xiaomi,redmi-router-ac2100)
-	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x20000"
+	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x10000"
+	ubootenv_add_uci_sys_config "/dev/mtd2" "0x0" "0x4000" "0x10000"
```

Comment:
Maybe secsize for nand devices needs to be set to 0x20000 (128KiB) ?

Reply:
@remittor why from
```
ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x20000"
```

to 

```
ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x1000" "0x10000"
```

Reply:
Because I don't want to break something on device `xiaomi,mi-router-4a-gigabit` (64 KiB sector)

Reply:
> Can't we just extend the switch and apply the correct sector based on the device to be extra safe?

OK. Let's do that. Now I'll try to fix it.